benches/benchmark_solver.rs

```diff
@@ -423,86 +423,86 @@ bench_lcgs!(
 // gossiping girls
 bench_lcgs!(
     gg1_circular,
```

Comment:
I renamed these to `ggN_circularQ` where `N` is the number of girls and `Q` is the query index. That way you can add `gg5_circularQ` to the bench as well.